website/source/docs/job-specification/template.html.md

```diff
@@ -39,8 +39,10 @@ job "docs" {
 }
 ```
 
-Nomad utilizes a tool called [Consul Template][ct]. For a full list of the
-API template functions, please refer to the [Consul Template README][ct].
+Nomad utilizes a tool called [Consul Template][ct]. The environment variables
+available in templates include [Nomad's runtime environment][env]. For a full
```

Comment:
`Since Nomad v0.5.3, the template can reference [Nomad's runtime environment variables][env]`